### DISCUSSION TITLE
Difference between pypi.org/project/sentry-sdk and pypi.org/project/sentry

### DISCUSSION THREAD

**Discussion by foobarna** [getsentry/sentry-python#2072] [Q&A] [answered]
Hello!

What is the difference between https://pypi.org/project/sentry-sdk/ and https://pypi.org/project/sentry/?

**Comment by antonpirker** [comment, marked as answer]
- https://pypi.org/project/sentry/ is the backend that powers Sentry.io (you can run all of Sentry by yourself see https://open.sentry.io/ )
- https://pypi.org/project/sentry-sdk/ is the package you install in your Python project to collect errors (and other data) and send it to Sentry.io
